doc/Community-Build/ProvisionReleaseBuild.md

```diff
@@ -0,0 +1,221 @@
+
+- All the prerequisites specified in the [Building the CORTX Environment for Single Node](Building-CORTX-From-Source-for-SingleNode.md) document must be satisfied.
+
+- The CORTX packages must be generated using the steps provided in the [Generate Cortx Build Stack guide](Generate-Cortx-Build-Stack.md).
+
+
+## Procedure
+
+1. Set repository URL using the following command:
+
+   ```
+   export CORTX_RELEASE_REPO="file:///var/artifacts/0"   
+   ```
+
+2. Run the following command to install the CORTX provisioner APIs and required packages:
+
+   ```
+   yum install -y yum-utils
+   yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/3rd_party/"
+   yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/cortx_iso/"
+
+   cat <<EOF >/etc/pip.conf
+   [global]
+   timeout: 60
+   index-url: $CORTX_RELEASE_REPO/python_deps/
+   trusted-host: $(echo $CORTX_RELEASE_REPO | awk -F '/' '{print $3}')
+   EOF
+
+   # Cortx Pre-requisites
+   yum install --nogpgcheck -y java-1.8.0-openjdk-headless
+   yum install --nogpgcheck -y python3 cortx-prereq sshpass
+
+   # Pre-reqs for Provisioner
+   yum install --nogpgcheck -y python36-m2crypto salt salt-master salt-minion
+
+   # Provisioner API
+   yum install --nogpgcheck -y python36-cortx-prvsnr
+
+   # Verify provisioner version (0.36.0 and above)
+   provisioner --version
+   ```
+
+3. Run the following commands to clean the temporary repos:
+
+   ```
+   rm -rf /etc/yum.repos.d/*3rd_party*.repo
+   rm -rf /etc/yum.repos.d/*cortx_iso*.repo
+   yum clean all
+   rm -rf /var/cache/yum/
+   rm -rf /etc/pip.conf
+   ```
+
+4. Create the config.ini file:
+
+     **Note:** The config.ini file requires you to add the metadata disk, data disk, and NICs, information. Run the following command to find the devices on your node:
+
+   ```
+   device_list=$(lsblk -nd -o NAME -e 11|grep -v sda|sed 's|sd|/dev/sd|g'|paste -s -d, -)
+   ```
+
+   A. To find the metadata disks value for storage.cvg.0.metadata_devices, run:
+
+      ```  
+      echo ${device_list%%,*}
+      ```
+
+   B. To find the data disks value for storage.cvg.0.data_devices, run:
+
+      ```
+      echo ${device_list#*,}
+      ```
+
+   C. To find the interfaces as per zones defined in your VM, run:
+
+      ```
+      firewall-cmd --get-active-zones
+      ```
+
+   D. Run the following command to create a config.ini file:
+
+      ```
+      vi ~/config.ini
+      ```
+
+   E. Paste the code below into the config file replacing your network interface names with ens33,ens34, ens35, and storage disks with /dev/sdb,../dev/sdc:
+      
+      **Note:** The values used in the below code are for example purpose, update the values as per the inputs received from the above steps.
+
+      ```
+      [srvnode_default]
+      network.data.private_interfaces=ens35
+      network.data.public_interfaces=ens34
+      network.mgmt.interfaces=ens33
+      storage.cvg.0.data_devices=/dev/sdb,/dev/sdc,/dev/sdd,/dev/sde,/dev/sdf,/dev/sdg,/dev/sdh,/dev/sdi
```

Comment:
Can we use `/dev/sdb` and metadata device and rest all as data devices. Just to inline with the configuration used in CI/CD

Reply:
If we use /dev/sdb as metadata device then we have to change in OVA R2 wiki as well as well as in OVA code so if this can be changed in CICD which will be one change

Reply:
Here also this is just a suggestion to make it aligned. This helps in troupbleshooting.